.travis.yml

```diff
@@ -53,6 +53,7 @@ jobs:
                 - ./check_trailing_spaces.sh || travis_terminate 1
                 - if [ -n "$COMMIT_SCA_FILES" ]; then ./dev-tools/vendor/bin/phpmd `echo "$COMMIT_SCA_FILES" | grep -Ev "^tests/Fixtures" | xargs | sed 's/ /,/g'` text phpmd.xml || travis_terminate 1; fi
                 - ./dev-tools/vendor/bin/composer-require-checker check composer.json --config-file=.composer-require-checker.json
+                - composer normalize -d ./dev-tools ./../composer.json --dry-run
```

Comment:
@keradus 

Added the `file` argument, and fixed the issue with the extra new lines in the diff, take a look at https://github.com/localheinz/composer-normalize/compare/0.5.0...0.6.0. Hope you like it!

Reply:
looks great ;)
actually, let us run it against both composer files, because "why not" :)

btw, why `composer/normalize` instead of `/vendor/bin/composer-normalize`? ah, as it's a plugin, not a "binary" app. that's why path was not there previously.